projects/sbb-esta/angular-business/tooltip/src/tooltip/tooltip-container.component.ts

```diff
@@ -0,0 +1,165 @@
+import { AnimationEvent } from '@angular/animations';
+import { ConnectionPositionPair } from '@angular/cdk/overlay';
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  HostBinding,
+  HostListener,
+  OnDestroy,
+  ViewEncapsulation
+} from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+
+import { sbbTooltipAnimations } from './tooltip-animations';
+
+/** Possible visibility states of a tooltip. */
+export type TooltipVisibility = 'initial' | 'visible' | 'hidden';
+
+/**
+ * Internal component that wraps the tooltip's content.
+ * @docs-private
+ */
+@Component({
+  selector: 'sbb-tooltip-container',
+  templateUrl: './tooltip-container.component.html',
+  styleUrls: ['./tooltip-container.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  animations: [sbbTooltipAnimations.tooltipState]
+})
+export class TooltipContainerComponent implements OnDestroy {
+  /**
+   * @docs-private
+   */
+  @HostBinding('attr.aria-hidden') ariaHidden = true;
+
+  /**
+   * Forces the element to have a layout in IE and Edge. This fixes issues where the element
+   * won't be rendered if the animations are disabled or there is no web animations polyfill.
+   * @docs-private
+   */
+  @HostBinding('style.zoom')
+  get styleZoom(): number {
+    return this._visibility === 'visible' ? 1 : null;
+  }
+
+  /** current connectionPositionPair of overlayRef */
+  connectionPositionPair: ConnectionPositionPair;
```

Comment:
This is an internal detail. Should be prefixed with _
Also comments start with upper case.